src/Plot.js

```diff
@@ -355,7 +355,7 @@ export default class Plot extends Viz {
         }
 
         // set axis title if not discrete
-        if (str !== k && (!this[`_${k}Config`].title || this[`_${k}Title`] === this[`_${k}Config`].title) && this._discrete !== k) {
+        if (str !== k && this[`_${k}Title`] === this[`_${k}Config`].title && this._discrete !== k) {
```

Comment:
This change actually doesn't fix the problem entirely, let me break down the if clause conditionals:
* `str !== k`: the user accessor string key is not the same as the axis itself (ie. "x" or "y")
* `(!this['_${k}Config'].title || this['_${k}Title'] === this['_${k}Config'].title)`: the user has not set the title in the config, or it was previously set by this function
* `this._discrete !== k`: it's not the discrete axis

By changing that 2nd condition to just "the title wasn't set previously by this function", that means that this works the first time whether or not the user has set the title, but on subsequent renders if the user changes the title it won't update. Try setting a timeout in your testing HTML file, or just playing with it in the browser console:

```js
var chart = new d3plus.Plot()
  .config(obj)
  .yConfig({title: "Custom Title"})
  .render();

setTimeout(() => {
  chart.yConfig({title: "Second Title"}).render();
}, 2000);
```

I think that all that needs to be changed is changing `!this['_${k}Config'].title` to something like `this['_${k}Config'].title === undefined`, which will cause this condition to fail if the user has set the value to `false`, but it will still work for the case of "the user has not defined a value".

Reply:
You're right, changing second condition works only on first visualization rendering.
I checked it using your timeout approach and by browser console and got your expected results.

Changing `!this['_${k}Config'].title` to `this['_${k}Config'].title === undefined` seems to be the solution. Second condition fails if user set value to `false` but still working on default or `undefined` status.

I added this change to the last commit.